packages/clay-shared/stories/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/* eslint-disable no-sparse-arrays */
+
+import '@clayui/css/lib/css/atlas.css';
+import {storiesOf} from '@storybook/react';
+import React from 'react';
+
+import {ClayPortal, FocusScope} from '../src';
+
+storiesOf('Components|FocusScope', module).add('default', () => {
```

Comment:
Hey @bryceosterhaus I wanted not to make this visible in our storybook, can we make sure that this story does not appear in production only in the development environment? This would really be good for exploratory testing.